client/blocks/term-form-dialog/index.jsx

```diff
@@ -264,28 +264,42 @@ class TermFormDialog extends Component {
 
 		return (
 			<FormFieldset>
-				<FormLegend>{ labels.parent_item }</FormLegend>
 				<FormLabel>
-					<FormCheckbox
-						ref="topLevel"
-						checked={ this.state.isTopLevel }
-						onChange={ this.onTopLevelChange }
-					/>
+					<FormToggle checked={ isTopLevel } onChange={ this.onTopLevelChange } />
 					<span>
-						{ translate( 'Top level', { context: 'Terms: New term being created is top level' } ) }
+						{ translate( 'Top level %(term)s', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
Indeed, please convert `context` to a `comment`, thanks!